.changeset/funny-shrimps-visit.md

```diff
@@ -0,0 +1,7 @@
+---
+'@sap-cloud-sdk/generator': minor
+'@sap-cloud-sdk/odata-v4': minor
+'@sap-cloud-sdk/odata-common': minor
```

Comment:
[q] Should I write `major` here? What is the correct heading for the PR then? `feat` needs a changeset with `minor`.

Reply:
Yes major is appropriate as this will be a breaking change and will be part of our next major release.